test/e2e/specs/wp-calypso-gutenberg-upgrade-spec.js

```diff
@@ -102,84 +96,11 @@ async function insertBlock( blockClass ) {
 	return blockInit && blockInit( block );
 }
 
-// Commented out for now because of ENOMEM erros on CI
-// /**
-//  * Take screenshot(s) of a given site's page.
-//  *
-//  * There's no way I know of to take a full-height screenshot of the page using wedriver. Because of
-//  * that, we basically scroll down the page in pieces and take screenshot of those areas. In the end
-//  * we have multiple screenshots that do represent the full page, vertically.
-//  *
-//  * @param { string } siteName the name of the WP test site (it will prepended to .wordpress.com)
-//  * @param { number } totalHeight The total height of the page
-//  * @param { number } viewportHeight The visible height
-//  * @param { Function } scrollCb A callback function that will be called to scroll down the page, it
-//  * should accept an integer that will be used in formula to calculate how much it should scroll.
-//  */
-// async function takeScreenshot( siteName, totalHeight, viewportHeight, scrollCb ) {
-// 	const now = Date.now() / 1000;
-
-// 	const siteScreenshotsDir = join( mediaHelper.screenshotsDir, siteName );
-// 	await fs
-// 		.access( siteScreenshotsDir )
-// 		.catch( () => fs.mkdir( siteScreenshotsDir, { recursive: true } ) );
-
-// 	for ( let i = 0; i <= totalHeight / viewportHeight; i++ ) {
-// 		await scrollCb( i );
-
-// 		const screenshotData = await driver.takeScreenshot();
-// 		const url = await driver.getCurrentUrl();
-// 		await mediaHelper.writeScreenshot(
-// 			screenshotData,
-// 			() => join( siteName, `${ siteName }-${ i }-${ now }` ),
-// 			{ url }
-// 		);
-// 	}
-// }
-
-// Commented out for now because of ENOMEM erros on CI
-// /**
-//  * Take screenshot(s) of the editor. Useful to diagnose glitches that might manifest visually and
-//  * that we still don't verify automatically.
-//  *
-//  * @param { string } siteName the name of the WP test site (it will prepended to .wordpress.com)
-//  */
-// async function takeEditorScreenshots( siteName ) {
-// 	const editorViewport = await driver.findElement(
-// 		By.css( 'div.interface-interface-skeleton__content' )
-// 	);
-
-// 	const editorViewportScrollHeight = await driver.executeScript(
-// 		'return arguments[0].scrollHeight',
-// 		editorViewport
-// 	);
-// 	const editorViewportClientHeight = await driver.executeScript(
-// 		'return arguments[0].clientHeight',
-// 		editorViewport
-// 	);
-
-// 	await takeScreenshot(
-// 		`${ siteName }-editor`,
-// 		editorViewportScrollHeight,
-// 		editorViewportClientHeight,
-// 		( i ) =>
-// 			driver.executeScript(
-// 				`arguments[0].scroll({top: arguments[0].clientHeight*${ i }})`,
-// 				editorViewport
-// 			)
-// 	);
-// }
-
 async function assertNoErrorInEditor() {
 	const errorDisplayed = await gEditorComponent.errorDisplayed();
 	assert.strictEqual( errorDisplayed, false, 'The block errored in the editor!' );
 }
 
-// Commented-out for now because of https://github.com/Automattic/jetpack/issues/16514
```

Comment:
I think we should re-activate the invalid block checks. The related issue is still open but it might worth double-checking to see if it's still failing.

Reply:
This has been done ✅